source/extensions/upstreams/tcp/BUILD

```diff
@@ -0,0 +1,25 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    # This is core Envoy config.
+    visibility = ["//visibility:public"],
```

Comment:
This triggers `This change appears to add visibility rules. Please get senior maintainer approval to add an exemption to check_visibility tools/code_format/check_format.py` and blocks ci

Could you help to move forward? @mattklein123 @alyssawilk 
Thanks

Reply:
I think this is fine as it maps to what we do for HTTP but I will defer to Alyssa who did the HTTP work related to this.

Reply:
Alyssa agrees on slack so I'm going to tweek `tools/code_format/config.yaml`